File: 3516-find-closest-person/3516-find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        var dx = Math.abs(z-x);
        var dy = Math.abs(z-y);

        if(dx == dy) return 0;
        else if(dx < dy) return 1;
        else return 2;
    }
}

File: 3516-find-closest-person/3516-find-closest-person.cs
public class Solution {
    public int FindClosest(int x, int y, int z) {
        var dx = Math.Abs(z-x);
        var dy = Math.Abs(z-y);

        if(dx == dy) return 0;
        else if(dx < dy) return 1;
        else return 2;
    }
}